app/code/Magento/GoogleAnalytics/Test/Unit/Observer/SetGoogleAnalyticsOnOrderSuccessPageViewObserverTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GoogleAnalytics\Test\Unit\Block;
```

Comment:
Namespace should be different. Please cherry-pick following commit:
https://github.com/magento/magento2/commit/6be5f32ad8aa25a164afe64ac4b174cdc84d8a61

Reply:
@ihor-sviziev Done.